app/src/main/java/com/bogdan/codeforceswatcher/features/auth/RestorePasswordMailSentComposeActivity.kt

```diff
@@ -0,0 +1,134 @@
+
+class RestorePasswordMailSentComposeActivity : ComponentActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            AlgoismeTheme {
+                RestorePasswordMailSentScreen()
+            }
+        }
+    }
+
+    @Composable
+    private fun RestorePasswordMailSentScreen() {
+
+        Box {
+            Scaffold(
+                modifier = Modifier.fillMaxSize(),
+                topBar = {
+                    NavigationBar { finish() }
+                },
+                bottomBar = {
+                    LinkText(
+                        linkTextData = listOf(
+                            LinkTextData(getString(R.string.check_your_spam_folder))
+                        ),
+                        modifier = Modifier
+                            .height(80.dp)
+                            .fillMaxWidth()
+                            .padding(horizontal = 20.dp),
+                        textStyle = MaterialTheme.typography.body1.copy(
+                            fontSize = 14.sp,
+                            color = MaterialTheme.colors.secondaryVariant
+                        ),
+                        paragraphStyle = ParagraphStyle(textAlign = TextAlign.Center)
+                    )
+                },
+                backgroundColor = MaterialTheme.colors.background
+            ) {
+                Column(
+                    modifier = Modifier.padding(horizontal = 20.dp),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+                    Spacer(Modifier.height(56.dp))
+
+                    Title(
+                        title = getString(R.string.check_your_box),
+                        style = MaterialTheme.typography.h3.copy(
+                            fontSize = 36.sp,
+                            textAlign = TextAlign.Center
+                        )
+                    )
+
+                    Spacer(Modifier.height(12.dp))
+
+                    Title(
+                        title = getString(R.string.open_mail_hint),
+                        style = MaterialTheme.typography.h6.copy(textAlign = TextAlign.Center)
+                    )
+
+                    Spacer(Modifier.height(72.dp))
+
+                    Icon(
+                        painter = painterResource(id = R.drawable.ic_email),
+                        contentDescription = "Email",
+                        tint = MaterialTheme.colors.onBackground
+                    )
+
+                    Spacer(Modifier.height(72.dp))
+
+                    AuthButton(getString(R.string.open_mail)) {
+                        startMailApp()
+                    }
+
+                    Spacer(Modifier.height(20.dp))
+
+                    AuthButton(
+                        label = getString(R.string.back_to_sign_in),
+                        modifier = Modifier.border(
+                            width = 2.dp,
```

Comment:
Isn't it `2.dp` in other places too? Then you can embed it in the component. Maybe there are other similarities.

Reply:
This way we will have borders on all buttons. So, sometimes buttons will have the same background and border color. Is it ok?

Reply:
Also, some buttons have a border width of `1.6.dp`

Reply:
Ah, OK, in this case never mind.